lib/cops/raise_message_cop.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# This cop checks if there is any "raise" code without a message
+# Examples:
+# Good -> raise "Message"
+# Good -> 
+module RuboCop
+    module Cop
+      module CustomCops
+        class RaiseMessageCop < Base
+          def on_send(node)
+            return unless node.command?(:raise)
+            # If the raise encountered does not have any arguments, add the offense
+            # It ensures that raise is always accompanied with an argument.
+            if node.arguments.size<1
```

Comment:
i guess raise without argument is valid inside an exception handler block. In that case, it will raise the exception which it just caught, can you verify it?

Reply:
For exception handler cases, raise will return a "RunTime" error if given without argument by default, if we want to raise an Exception, we need to pass it as an argument.

With no arguments, raises the exception in $! or raises a RuntimeError if $! is nil.

https://stackoverflow.com/questions/4800698/what-is-the-difference-between-raise-foo-and-raise-exception-newfoo

Reply:
I was talking about this case: https://stackoverflow.com/a/23771227